inductiva/resources/machine_cluster.py

```diff
@@ -55,6 +55,9 @@ def from_api_response(cls, resp: dict):
         machine_group.register = False
         return machine_group
 
+    def __repr__(self):
+        return f"MPI Cluster: {self.name}"
+
```

Comment:
these dunder methods should use __str__ and not __repr__.
The latter is used when you want to rebuild the object, whereas the former should be used when you want to have a human readable representation of the object.
Ex:

```
repr(x) -> MPICluster(165442829)

vs

str(x) -> MPI Cluster with 6 nodes and id=165442829
```

Reply:
The issue opened referred to the `get` method of machines, see here: https://github.com/inductiva/tasks/issues/48.

So I though to kill two birds with one stone. For now, I will  use `__str__` and leave that issue open either way.
But I will keep the simple message for now.